packages/vm/test/t8n/ethereumjs-t8ntool.sh

```diff
@@ -5,4 +5,4 @@ if [[ "$1" == "--version" ]]; then
 fi
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 export NODE_OPTIONS="--max-old-space-size=4096" 
-npx tsx "$SCRIPT_DIR/launchT8N.ts" "$@"
\ No newline at end of file
+npx tsx --conditions=typescript "$SCRIPT_DIR/launchT8N.ts" "$@"
```

Comment:
Do we want to keep this flag?

Reply:
I like it since most of your edits will be in `evm` but this lives in the `vm`.  Otherwise, you have to rebuild `evm` every time.

Reply:
Ah yes sorry my question should have gotten more context: this will thus only work on node versions where this flag is available. I'm not sure about this one, this means that people need to have node v22 installed (right?)

Reply:
No, the `conditions` flag was introduced in either 18 or 20

Reply:
Ok, lets keep! :smile: :+1: